src/OrchardCore/OrchardCore.Localization.Core/Extensions/CultureDictionaryExtensions.cs

```diff
@@ -0,0 +1,23 @@
+using System.Linq;
+
+namespace OrchardCore.Localization.Extensions
+{
+    internal static class CultureDictionaryExtensions
+    {
+        public static string GetTranslationWithIgnoreContext(this CultureDictionary cultureDictionary, string key)
```

Comment:
May be it's bad idea to ignore the `context` but this is the way to come up with to get the data annotations message if there is, because the context all the time will be `OrchardCore.Localization.DataAnnotations.LocalizedDataAnnotationsMvcOptions`

@jtkech @sebastienros if there's a better way, please shoot it to me

Reply:
I made a comment below where you use it